src/middlewares/AutoEnsure.ts

```diff
@@ -1,84 +1,81 @@
-import { ApplyMiddlewareOptions } from '../lib/decorators/ApplyMiddlewareOptions';
+import { ApplyMiddlewareOptions } from '../lib/decorators';
 import { Middleware } from '../lib/structures/Middleware';
-import { BuiltInMiddleware, Method, Payloads } from '../lib/types';
+import { Method, Payloads } from '../lib/types';
 
 @ApplyMiddlewareOptions({
-  name: BuiltInMiddleware.AutoEnsure,
+  name: 'autoEnsure',
   position: 0,
```

Comment:
What does position do?

Reply:
Tells Josh what order to run middleware in